source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/NumberInputRenderer.java

```diff
@@ -77,6 +77,6 @@ else if ((numberInput = NumberInput.dynamic_cast(baseCardElement)) == null)
 
         return editText;
     }
-    
```

Comment:
Any idea what's causing this extra spacing to be added?

Reply:
It's changing from

```
<spaces><linefeed>
```

to 

```
<linefeed>
```